recipe/meta.yaml

```
@@ -1,55 +1,59 @@
 {% set name = "pycaret" %}
-{% set version = "3.0.2" %}
+{% set version = "3.3.2" %}
 
 package:
   name: {{ name|lower }}-suite
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
-  sha256: d1bea16403ecf848720aec53d2bccf39374025e8ae3232321b173060489aca73
+  sha256: aa89ed0a5795e05200e0e3d44ec3ee96bfea499d5db2df5e37ce903a5bff6ace
 
 build:
-  number: 1
-  # Not available on linux-s390x: kaleido-core, plotly-resampler, sktime, fugue, pyod, numba, moto
-  # kaleido-core is not available on linux-ppc64le
-  skip: true  # [py<38 or (linux and (s390x or ppc64le))]
+  number: 0
+  # RuntimeError: ('Pycaret only supports python 3.9, 3.10, 3.11. Your actual Python version: ', sys.version_info(major=3, minor=12, micro=9, releaselevel='final', serial=0), 'Please DOWNGRADE your Python version.')
+  skip: true  # [py<39 or py>311]
+  # scikit-learn requires incompatible scipy due to some bug:
+    # Build fails on osx-64 without the newest version of scipy.
+    # https://github.com/scikit-learn/scikit-learn/issues/28367#issuecomment-1941101095
+    #- scipy >=1.13.0  # [osx and x86_64]
+  # while pycaret needs <=1.11.4 for compatibility (verified)
+  skip: true  # [osx and x86_64]
 
 requirements:
   host:
     - python
-    - numpy {{ numpy }}
+    - numpy >=1.21, <1.27
```

```suggestion
    - numpy >=1.21,<1.27
```
A redundant whitespace and it's repeated below

---

Done